R/Layer.R

```diff
@@ -0,0 +1,110 @@
+#' Create a custom Layer
+#' 
+#' @param classname the name of the custom Layer.
+#' @param defs definition of methods.
```

Comment:
Update with parameter documentation for `initialize`, `build`, `call`.

Reply:
I assume that if the method `compute_output_shape` needs to be specified, that can happen through the `...`, correct?
I'm curious about the reason for explicitly listing the `build` method as one of the parameters for `Layer`. Is that for documentation, because it is more likely that it will need to be specified?
It might be nice to also list `compute_output_shape` explicitly, and even the less-commonly used methods such as `compute_mask`.

Reply:
I added `build` just for the probability for it to be specified. I can add `compute_output_shape` too. Any other method can be defined using the ellipsis.